package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@flowio/redux-fetch",
-  "version": "0.14.1",
```

Comment:
No need to bump package version. Generally, I run `npm version` before publishing the package. This has the potential of assigning the wrong tag.